src/library/DIPS.Mobile.UI/Components/Chips/Android/ChipHandler.cs

```diff
@@ -121,4 +128,48 @@ private static int[][] CreateColorStates(Color color, out int[] colors)
     {
         handler.PlatformView.ChipStrokeWidth = (float) chip.BorderWidth;
     }
+    
+    private static partial void MapTitleColor(ChipHandler handler, Chip chip)
+    {
+        if (chip.TitleColor is null) return;
+        handler.PlatformView.SetTextColor(chip.TitleColor.ToPlatform());
+        handler.PlatformView.CheckedIconTint = new ColorStateList(CreateColorStates(handler.VirtualView.TitleColor, out var colors), colors);
+    }
+
+    private static partial void MapIsToggleable(ChipHandler handler, Chip chip)
+    {
+        if (!handler.VirtualView.IsToggleable)
+            return;
+        
+        handler.PlatformView.Checkable = true;
+        DUI.TryGetResourceId(Icons.GetIconName(handler.ToggledIconName), out var id, defType:"drawable");
+        if (id is not 0)
+        {
+            var drawable = Platform.AppContext.Resources?.GetDrawable(id);
+            handler.PlatformView.CheckedIcon = drawable;
+        }
+        handler.PlatformView.SetOnCheckedChangeListener(new OnToggledChangedListener(handler));
+    }
+    
+    private static partial void MapIsToggled(ChipHandler handler, Chip chip)
+    {
+        //Make sure not to mess with close button
+        if (handler.VirtualView.HasCloseButton)
+            return;
+
+        if (!handler.VirtualView.IsToggleable)
+            return;
+        
+        if (handler.VirtualView.IsToggled)
```

Comment:
Kanskje forenkle koden med å sette `handler.PlatformView.Checked / CheckedIconVisible=handler.VirtualView.IsToggled` for å minske noen linjer